internal/ingress/controller/handlers/endpoints.go

```diff
@@ -48,23 +49,42 @@ func (h *EnqueueRequestsForEndpointsEvent) Delete(e event.DeleteEvent, queue wor
 func (h *EnqueueRequestsForEndpointsEvent) Generic(event.GenericEvent, workqueue.RateLimitingInterface) {
 }
 
-//TODO: this can be further optimized to only included ingresses referenced this endpoints(service) :D
+//TODO: this can be further optimized to only reconcile the target group referenced by the endpoints(service) :D
```

Comment:
The comment should be removed as this is implemented now.

Reply:
Hi, there is still a way to optimize this function. Currently, we reconcile the whole ingress, we could only reconcile the target group associate with the backend.